pages/declaration.js

```diff
@@ -45,20 +45,24 @@ class Declaration extends React.Component {
           ref="submitRegistration"
           csrfToken={this.props.csrfToken}
         >
-          <BackButton {...this.props} />
+          <BackButton {...this.props} t={this.props.t} />
           <ProcessedErrorSummary
+            t={this.props.t}
             validatorErrors={this.props.validatorErrors}
             onHandleErrorClick={OnHandleErrorClick}
           />
           <SubmissionErrorSummary
+            t={this.props.t}
```

Comment:
Don't believe this needs props.t either, similar situation to Processed error summary